packages/flipper-react/src/Flipper.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactNode } from "react";
+import cn from "classnames";
+
+export const Flipper: React.FC<{
+    showFirst: boolean;
```

Comment:
Føler navngivingen her kan bli litt uklar. Hva med primary/secondary, eller initial/secondary og en prop for `showSecondary`?